### DISCUSSION TITLE
svgs and the web

### DISCUSSION THREAD

**Discussion by scythewyvern** [nuejs/nue#210] [Q&A]
my question is not directly related to nue but still, it seems to me that this is a good place for discussion.
so svg. what's the best way to use svg on the web? on the nuejs site svg are used in the background image and this seems like a good solution, but I'm wondering how else I can use them?

sometimes you need to use a lot of svg logos, different sizes and colors, and sometimes it's just icons, it's very difficult...

**Comment by SergioKnows** [comment]
sometimes i like to use svg because are easy to edit using some programs like corel draw

**Comment by tipiirai** [comment]
I think there is only one use case when inlining makes sense: dynamic generation and customization. Otherwise, background images work better, because they keep the HTML structure clean and absent from presentational elements. 

Thoughts?
